build.gradle

```diff
@@ -89,6 +89,7 @@ allprojects {
         httpclientVersion = '4.5.13'
         httpcoreVersion = '4.4.15'
         jacksonVersion = '2.13.3'
```

Comment:
Since we're updating databind to 2.13.3, I wonder if we should move the other jackson libs close - ie to 2.13.4 ?

Reply:
Yes makes sense.